pkg/apis/stable/v1alpha1/common.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+// Block of const Error messages
+const (
+	ErrContainerRequired = "Container is required when using multiple containers in the pod template"
+	ErrHostPortDynamic   = "HostPort cannot be specified with a Dynamic PortPolicy"
+	ErrPortPolicyStatic  = "PortPolicy must be Static"
+)
+
+// crdName is an interface to get Name and Kind of CRD
+type crdName interface {
```

Comment:
```suggestion
type crd interface {
```

Maybe? :man_shrugging:

Reply:
Good naming for this small file, applied propose changes